packages/palette/src/elements/BarChart/BarChart.story.tsx

```diff
@@ -342,5 +342,5 @@ storiesOf("Components/BarChart", module)
         </Box>
       )
     },
-    { chromatic: { delay: 500, diffThreshold: 0.2 } }
```

Comment:
@damassi why were all the charts made to be more sensitive?

Reply:
Lol are they? Not sure 🤦

Reply:
😆 https://www.chromatic.com/docs/threshold

> Chromatic’s default threshold is .063 which balances high visual accuracy with low false positives (for example, from artifacts like anti-aliasing). 0 is the most accurate. 1 is the least accurate.

I think all of the places where these were failing weren't due to the diffThreshold but rather views animating in. (So like the bar chart rising, the modal moving into the frame, etc)